changed test points/jumpers, some other changes

aidan-9880 opened the pull request:
Changed JTAG jumpers to solder jumpers, I2C test points to pads (#37) 
Added a photoresistor connected to pin 4, uses ADC1 in the ESP32 (half of #29) 
Also made some minor changes to the silkscreen, mostly text settings/position

jdbaptista commented:
I don't see an LCSC part number for the photoresistor in the JLC plugin; would you be able to add that in or reference it here? I would like to do some analysis into that to make sure things will work well when it's time to implement that functionality.

Also, to remove AC noise on ADC readings a capacitor should be added as briefly discussed in the ['Minimize Noise' section](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/peripherals/adc_calibration.html#adc-kconfig-options) of the documentation. I think [adding a somewhat large one directly to the PHOTO net and bringing it to ground](https://www.electronicshub.org/bypass-capacitor-tutorial/#:~:text=Bypass%20capacitors%20are%20used%20to,the%20loudness%20of%20the%20signal.) should be fine because we won't be sampling often and ambient light doesn't change quickly.

aidan-9880 commented:
Added photoresistor part number and 100nF bypass cap
Changed JTAG test points single-sided and moved to the back
Added RGB OTA indicator LED , next to other status LEDs (#43)
Added traffic level key, below compass rose (#32)

jdbaptista commented:
nice looks great
